features/mbedtls/importer/adjust-config.sh

```diff
@@ -66,7 +66,8 @@ add_code
     "#endif\n"                                                                        \
     "\n"                                                                              \
     "#if defined(MBEDTLS_SSL_TLS_C) && !defined(MBEDTLS_TEST_NULL_ENTROPY) && \\\\\n" \
-    "    !defined(MBEDTLS_ENTROPY_HARDWARE_ALT)\n"                                    \
+    "    !defined(MBEDTLS_ENTROPY_HARDWARE_ALT) && \\\\\n"                            \
```

Comment:
We don't need so many backslashes here, do we?

Reply:
Without them we barely make it at the line being 79 characters wide. I will change this.

Reply:
Four ```\\\\```s are to retain one ```\``` used as line continuation escape character in multi line macro.

Reply:
That's understood, but we didn't have them before. I'm happy with the currently proposed change.

Reply:
They were introduced because we were extending the macro with another line. 
Now the change is made in the same line. Hence not needed.